cmake/setup.cmake

```diff
@@ -8,7 +8,7 @@ set(__cmake_setup_INCLUDED 1)
 set(CMAKE_CXX_FLAGS "-m64 ${CMAKE_CXX_FLAGS}")
 
 # Enable C++14 and link libc++
-set(CMAKE_CXX_FLAGS "-ferror-limit=3 ${CMAKE_CXX_FLAGS} -std=c++1y -stdlib=libc++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++1y -stdlib=libc++")
```

Comment:
Sorry about that. I had a bit of trouble getting ti to build on my system
because of that flag. Reverted that change.

On Thu, Jun 4, 2015 at 3:45 PM, Berkus Decker <notifications@github.com>
wrote:

> In cmake/setup.cmake
> <https://github.com/berkus/dir_monitor/pull/9#discussion_r31757776>:
>
> > @@ -8,7 +8,7 @@ set(__cmake_setup_INCLUDED 1)
> >  set(CMAKE_CXX_FLAGS "-m64 ${CMAKE_CXX_FLAGS}")
> >
> >  # Enable C++14 and link libc++
> > -set(CMAKE_CXX_FLAGS "-ferror-limit=3 ${CMAKE_CXX_FLAGS} -std=c++1y -stdlib=libc++")
> > +set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++1y -stdlib=libc++")
>
> Doesn't look like line-ending change to me. Explain?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/berkus/dir_monitor/pull/9/files#r31757776>.
>